templates/cluster/vsphere-standalone-cp/values.yaml

```diff
@@ -10,6 +10,8 @@ clusterNetwork:
     cidrBlocks:
     - "10.96.0.0/12"
 
+clusterLabels:
```

Comment:
i am not sure but isn't that better (more explicit) to set the `{}` instead of nothing to set an explicit type of the object? the question is applicable for the rest of the charts